bin/init.sh

```diff
@@ -68,16 +68,16 @@ fi
 PROXYVERSION=$(grep envoy-debug pilot/docker/Dockerfile.proxy_debug  |cut -d: -f2)
 PROXY=debug-$PROXYVERSION
 
-# Save envoy in vendor, which is cached
-if [ ! -f vendor/envoy-$PROXYVERSION ] ; then
+# Save envoy in $out
+if [ ! -f $OUT/envoy-$PROXYVERSION ] ; then
     mkdir -p $OUT
     pushd $OUT
     # New version of envoy downloaded. Save it to cache, and clean any old version.
 
     DOWNLOAD_COMMAND=""
     if command -v curl > /dev/null; then
-       if curl --version | grep Protocols  | grep https; then
-	   DOWNLOAD_COMMAND='curl -Lo -'
+       if curl --version | grep Protocols  | grep https > /dev/null; then
+	   DOWNLOAD_COMMAND='curl -Lso -'
```

Comment:
add -f while at it (#3231)

Reply:
I'll leave it for a specific fix, not sure how it interact with the wget ( or why we even have wget ).

Reply:
Also there is a tar extraction, cleanup - seems better as a separate PR.

Reply:
if you're changing the curl command options please add f to curl options / why not?

Reply:
I'm changing the curl command to make it less verbose. 
I don't think -f should be used, and even if it is - I don't think it's reasonable to ask to add unrelated features/fixes to a PR.

Reply:
( -f would need to be tested, logic may need to be changed, there is also the wget stuff around - I don't want to add that complexity to this PR, and I don't think it's a P0)